File: 74751_lab1_task1.py
# 74751_lab1_task1.py
distance = float(input("Enter distance between ECG and PPG sensors (meters): "))
time = float(input("Enter time delay (seconds): "))
pwv = distance / time
sbp = 120 + (pwv - 5) * 10
print(f"Pulse Wave Velocity (PWV): {pwv}")
print(f"Systolic Blood Pressure (SBP): {sbp}")

File: 74751_lab1_task1.cpp
// 74751_lab1_task1.cpp
#include <iostream>

using namespace std;

int main() {
    double distance, time;
    cout << "Enter distance between ECG and PPG sensors (meters): ";
    cin >> distance;
    cout << "Enter time delay (seconds): ";
    cin >> time;
    
    double pwv = distance / time;
    double sbp = 120 + (pwv - 5) * 10;
    
    cout << "Pulse Wave Velocity (PWV): " << pwv << endl;
    cout << "Systolic Blood Pressure (SBP): " << sbp << endl;
    
    return 0;
}
